lib/core-integration/src/Test/Integration/Faucet.hs

```diff
@@ -1614,12 +1614,19 @@ shelleyIntegrationTestFunds :: [(Address, Coin)]
 shelleyIntegrationTestFunds = mconcat
     [ seqMnemonics >>= (take 10 . map (, defaultAmt) . addresses . SomeMnemonic)
 
-    , zip
-        (addresses $ SomeMnemonic onlyDustWallet)
-        (map Coin [1,1,5,12,1,5,3,10,2,3])
+    , zip (addresses $ SomeMnemonic onlyDustWallet) (map Coin [ 1_000_000
+                                               , 1_000_000
+                                               , 5_000_000
+                                               , 12_000_000
+                                               , 1_000_000
+                                               , 5_000_000
+                                               , 3_000_000
+                                               , 10_000_000
+                                               , 2_000_000
+                                               , 3_000_000 ])
```

Comment:
That's some interesting choice of indentation we have here.

Reply:
:no_mouth: